Packages/KeyAppKit/Package.swift

```diff
@@ -111,12 +111,22 @@ let package = Package(
         .package(url: "https://github.com/p2p-org/FeeRelayerSwift", branch: "feature/simple-topup-and-sign"),
         .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", .upToNextMajor(from: "1.6.0")),
         .package(url: "https://github.com/Boilertalk/Web3.swift.git", from: "0.6.0"),
-        .package(url: "https://github.com/trustwallet/wallet-core", branch: "master"),
+        // .package(url: "https://github.com/trustwallet/wallet-core", branch: "master"),
```

Comment:
why did it change?

Reply:
@TrGiLong can you explain?

Reply:
We replace with direct using ``WalletCore.xcframework.zip`` and ``SwiftProtobuf.xcframework.zip``

There is conflict with Google service due using ``SwiftProtobuf``